framework/test/bundles/libH/TestBundleH.cpp

```diff
@@ -66,19 +67,20 @@ class TestProduct2 : public TestProduct, public TestBundleH2
 class TestBundleHPrototypeServiceFactory : public PrototypeServiceFactory
```

Comment:
excellent idea. I'm going to take a little time to do this and see if I can break the framework.